packages/app-project/src/screens/ProjectHomePage/ProjectHomePage.js

```diff
@@ -10,33 +10,49 @@ import ProjectStatistics from '../../shared/components/ProjectStatistics'
 import ZooniverseTalk from './components/ZooniverseTalk'
 import ThemeModeToggle from '../../components/ThemeModeToggle'
 import ProjectAnnouncement from './components/ProjectAnnouncement'
+import { Media } from '../../shared/components/Media'
 
 function ProjectHomePage (props) {
   const { screenSize } = props
-  const responsiveColumns = (screenSize === 'small') ? ['auto'] : ['auto', '1em']
   return (
-    <>
+    <Box align='center'>
       <ProjectAnnouncement />
       <Hero />
-      <Grid gap='medium' margin='medium'>
-        <Grid columns={responsiveColumns} gap='small'>
-          <ZooniverseTalk />
+
+      <Media at='default'>
```

Comment:
Heads up that I found, while working on the classify page, that this inserts an empty div into the DOM, which can then break the layout in the storybook. `fresnel` styles these empty divs with `display: none` in the SSr'ed page, but that doesn't happen in the storybook.

Reply:
Thanks - I'm okay with it here, since it's a page-level component, but I'd like to solve this for the storybook at some point .

Reply:
Yeah. I think it's only a big problem inside grids, where those extra elements get rendered as empty columns or rows. I can see why `fresnel` have done this, but it's a bit of a hack when you only want one of those `Media` components to render, and the other to output `null`.